app/views/devise/passwords/new.html.erb

```diff
@@ -1,16 +1,39 @@
 <%= render "devise/shared/error_messages", resource: resource %>
 
-<div class="mt-8 mb-10 crayons-card p-7 pt-3 align-left mx-auto" style="max-width:490px;">
-  <h2 class="pb-4">Forgot your password?</h2>
-  <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f| %>
+<div class="js-new-password flex flex-col items-center s:mx-auto mx-4" style="max-width:488px;">
+  <% if Settings::General.resized_logo.present? %>
+    <a class="mt-6" href="/">
+      <img class="registration__logo" src="<%= Settings::General.original_logo %>" alt="<%= community_name %>">
+    </a>
+  <% end %>
 
-    <div class="crayons-field">
-      <%= f.label :email, class: "crayons-field__label" %>
-      <%= f.email_field :email, class: "crayons-textfield", placeholder: "you@email.com", autofocus: true, required: true %>
-    </div>
+  <h2 class="crayons-title s:mt-6 mt-4">Forgot your password?</h2>
+  <p class="crayons-subtitle-3 fw-normal mt-1 color-secondary align-center ">Enter the email address associated with your account, and we'll send you a link to reset your password.</p>
```

Comment:
Non blocking: Is this something we’d want to abstract into yml for internationalization?

Reply:
+1 from me, I think we should be reducing the use of bare text as much as possible even if we don't actually translate it yet

Reply:
Thanks. I have converted both all the strings from both the files to internationalisation. 
Currently there is not `fr` supported file for the entire file in which these translations have been added so not adding the french translations.